### PR TITLE
fix(am): docs + accesslog.path fix

### PR DESCRIPTION
**Pull request by remisultan**
**Issue**

https://github.com/gravitee-io/issues/issues/7479

**Description**

Added documentation + fix for accesslog.path 

**Additional context**

https://github.com/gravitee-io/helm-charts/pull/238

### CODE REVIEW

File: am/README.adoc [hunk 1]
```diff
@@ -183,6 +183,34 @@ replicaset] for detailed documentation on helm chart.
...
+
+|api.http.responseHeaderSize |int |`+8192+` |
+
+|api.http.pool.minThreads |int |`+8192+` |
+
+|api.http.pool.maxThreads |int |`+8192+` |
+
+|api.http.pool.idleTimeout |int |`+8192+` |
+
+|api.http.pool.queueSize |int |`+8192+` |
```
Comment (jhaeyaert) [inline am/README.adoc:208]: I'm pretty sure that these are not the real default values 🙂

File: am/README.adoc [hunk 2]
```diff
@@ -183,6 +183,34 @@ replicaset] for detailed documentation on helm chart.
...
+
+|api.http.pool.accesslog.enabled |boolean |`+true+` |
+
+|api.http.pool.accesslog.path |string |`++` |
```
Comment (jhaeyaert) [inline am/README.adoc:212]: `++` as default log path ?

File: am/templates/api/api-configmap.yaml [hunk 3]
```diff
@@ -32,7 +32,7 @@ data:
       {{- if .Values.api.http.accesslog }}
       accesslog:
         enabled: {{ .Values.api.http.accesslog.enabled | default true }}
-        {{- if .Values.api.http.path }}
+        {{- if .Values.api.http.accesslog.path }}
```
Comment (jhaeyaert) [inline am/templates/api/api-configmap.yaml:35]: 👍

**Review by jhaeyaert — CHANGES_REQUESTED** [review]

File: am/README.adoc (hunk 1 above)
Comment (remisultan) [reply am/README.adoc]: FP

**Review by jhaeyaert — APPROVED** [review]

**Pull request merged**